app/uk/gov/hmrc/taxhistory/services/EmploymentHistoryService.scala

```diff
@@ -227,9 +227,9 @@ class EmploymentHistoryService @Inject()(
     }
   }
 
-  /*
-  A convenience method used when auditing.
-   */
+  /**
+    * A convenience method used when auditing.
+    */
```

Comment:
For private methods I usually don't put Scaladoc as nobody outside the class would see it, I just use plain comments, but it won't do any harm either.

Reply:
Sure, wasn't sure if this was on purpose or just missed a *